src/locutus/model/terminology.py

```diff
@@ -499,8 +563,8 @@ def set_mapping(self, code, codings, editor):
 
         old_mappings = ""
         try:
-            assert code is not None
-            mapping = tmref.document(code).get().to_dict()
+            assert code_index is not None

```

Comment:
I don't want to suggest you replace asserts yet, but I would like to verify that they do result in a 400+ error and that the application just aborts the call to the client as opposed to forcing the server to restart. I suspect that is true, but I worry about about those assertions, since they would normally just kill the running process.